godot-core/src/registry/property/phantom_var.rs

```diff
@@ -135,6 +135,12 @@ impl<T: GodotType + Var> Hash for PhantomVar<T> {
     fn hash<H: Hasher>(&self, _state: &mut H) {}
 }
 
+/// SAFETY: This type contains no data.
+unsafe impl<T: GodotType + Var> Send for PhantomVar<T> {}
+
+/// SAFETY: This type contains no data.
+unsafe impl<T: GodotType + Var> Sync for PhantomVar<T> {}
```

Comment:
Nitpick: comment, does not need to be part of public docs :slightly_smiling_face: 
```suggestion
// SAFETY: This type contains no data.
unsafe impl<T: GodotType + Var> Send for PhantomVar<T> {}

// SAFETY: This type contains no data.
unsafe impl<T: GodotType + Var> Sync for PhantomVar<T> {}
```

Reply:
Oops.